File: app/src/main/java/com/example/ilkapk/MainActivity.java
package com.example.ilkapk;

import android.app.Activity;
import android.graphics.Color;
import android.os.Bundle;
import android.view.Gravity;
import android.widget.TextView;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        TextView text = new TextView(this);
        text.setText("G");
        text.setTextSize(120);
        text.setTextColor(Color.YELLOW);
        text.setGravity(Gravity.CENTER);

        setContentView(text);
    }
}

File: app/src/main/java/com/example/ilkapk/MainActivity.kt
package com.example.ilkapk

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import android.graphics.Color
import android.view.Gravity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val text = TextView(this)
        text.text = "G"
        text.textSize = 96f
        text.setTextColor(Color.YELLOW)
        text.gravity = Gravity.CENTER

        setContentView(text)
    }
}

